mne/io/eeglab/eeglab.py

```diff
@@ -443,3 +518,54 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             reject=reject, flat=flat, reject_tmin=reject_tmin,
             reject_tmax=reject_tmax, add_eeg_ref=False, verbose=verbose)
         logger.info('Ready.')
+
+
+def _read_eeglab_events(eeg, event_id=dict(), event_id_func=lambda t:
+                        int("".join([x for x in t if x.isdigit()]))):
```

Comment:
I'm inclined towards having a docstring which says: `callable | None` with `None` as the default ... but let's see what the others say